templates/feature/edit.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}{{ 'update_feature'|trans }}{% endblock %}
+
+{% block body %}
+    {{ include('feature/components/_back_to_list_button.html.twig') }}
+    <h1>{{ 'update_feature'|trans }}</h1>
```

Comment:
We could maybee have this h1 directly using the content of title in the parent  template